src/main/scala/djinni/Main.scala

```diff
@@ -308,7 +308,7 @@ object Main {
       None
     }
     val objcSwiftBridgingHeaderWriter = if (objcSwiftBridgingHeaderName.isDefined && objcOutFolder.isDefined) {
-      val objcSwiftBridgingHeaderFile = new File(objcOutFolder.get.getPath, objcSwiftBridgingHeaderName.get + ".h")
+      val objcSwiftBridgingHeaderFile = new File(objcHeaderOutFolder.get.getPath, objcSwiftBridgingHeaderName.get + ".h")
```

Comment:
dumb question from me: Wouldn't it be even cooler if one could not only specify the name of the bridging header but the full path? 🤔

Reply:
This is an absolute valid question. 

There is a/some relation between the bridging / umbrella header , and the public Objective-C included files.
The bridging header must find them, and as it is now, the exiting implementation will do so wit this solution.

I see also no value to put them into a different place, since this will only make everything more complex for no, to me visible, benefit.